public/stylesheet.css

```diff
@@ -178,6 +188,8 @@ body.light {
   --color-button-text-hover: var(--lm-color-button-text-hover);
   --color-disabled: var(--lm-color-disabled);
   --color-grid-item-divider: var(--lm-color-grid-item-divider);
+  --color-menu-icon: var(--lm-color-menu-icon);
```

Comment:
I think we can and should just reuse the `--color-button` variables for the "menu" item. If you want to have a separate variable - e.g. because we may want a different color in the header at some point - then this variable should for now reference the existing one to ensure they're the same as long as that's the goal.

Also, "menu" is a bit of an odd choice of word as it doesn't appear in the HTML.

Reply:
removed color-menu-icon, reused color-button-active